src/atoms/heading/Heading.scss

```diff
@@ -1,17 +1,22 @@
 @import '../../../assets/styles/_globals.scss';
 
-$headings-margin                : 0 0 0.5em 0 !default;
-$headings-font-family           : $font-family-base !default;
-$headings-font-weight           : 700 !default;
-$headings-line-height           : $font-line-height !default;
-$headings-color                 : $color-secondary !default;
-$headings-text-transform        : none !default;
-$heading-font-size--first-level : 18px !default;
-$heading-font-size--second-level: 18px !default;
-$heading-font-size--third-level : 14px !default;
-$heading-font-size--fourth-level: 14px !default;
-$heading-font-size--fifth-level : 12px !default;
-$heading-font-size--sixth-level : 12px !default;
+$headings-margin                          : 0 0 0.5em 0 !default;
```

Comment:
`heading__margin`
It's old style of writting variables, please check e.g button component